src/main/java/com/matt/forgehax/mods/NoRender.java

```diff
@@ -0,0 +1,33 @@
+package com.matt.forgehax.mods;
+
+import com.matt.forgehax.events.Render2DEvent;
+import com.matt.forgehax.util.mod.Category;
+import com.matt.forgehax.util.mod.ToggleMod;
+import com.matt.forgehax.util.mod.loader.RegisterMod;
+import net.minecraft.entity.item.EntityItem;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import static com.matt.forgehax.Helper.getWorld;
+
+@RegisterMod
+public class NoRender extends ToggleMod {
+
+    public NoRender() {
+        super(Category.RENDER, "NoRender", false, "Stops rendering items on ground");
+    }
+
+    @SubscribeEvent
+    public void onRender2D(Render2DEvent event){
```

Comment:
I believe there is a forge event called everytime an entity is rendered, and you can cancel it. Not sure if it's better than what you're doing here but if it worked it would be faster than iterating over all existing entities every frame.

Reply:
Interesting, didn't know that, however i think that just hides the entity (while still being in the entitylist) while this removes it entirely which will cause less lag. Can add a setting to either hide / remove

Reply:
Sorry about the misunderstanding. I thought dead entities would remain in the loaded entity list. Just change the event to ClientTickEvent and use event.getPhase to only execute this code in the PRE phase.